stacks/jhipster-online/2.33.0/devfile.yaml

```diff
@@ -0,0 +1,94 @@
+schemaVersion: 2.2.2
+metadata:
+  name: jhipster-online
+  version: 2.33.0
+  description: Stack with the JHipster Online on DevSpaces Red Hat OpenShift
+  displayName: JHipster Online
+  icon: https://raw.githubusercontent.com/maximilianoPizarro/ecommerce-oracle/main/jhipster-icon.png
+  website: https://start.jhipster.tech
+  tags:
+    - Java
+    - JHipster
+    - Angular
+    - Spring
+  versions:
+    - version: 2.33.0
+      default: true
```

Comment:
I don't believe we need to specifiy version # and default here, just `version: 2.33.0` would suffice.

Ref: https://github.com/devfile/registry/blob/main/stacks/go/2.1.0/devfile.yaml#L13

Reply:
Hi! Okey, the change on this [commit](https://github.com/devfile/registry/pull/480/commits/9443741521f808f2fd3bb259c55902746be0e160).